### PR TITLE
craftable union weapon update/sprite changes

### PR DESCRIPTION
**Pull request by thezblah**
Changes a few underused weapons craftable by union to be more unique.  Changes include sprites, names, and descriptions.  Weapon stats are unchanged.

### CODE REVIEW

File: code/game/objects/items/weapons/tools/simple_weapons.dm [hunk 1]
```diff
@@ -258,10 +258,10 @@
 	alt_mode_lossrate = 0.3
 
 /obj/item/tool/sword/katana
-	name = "katana"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/simple_weapons.dm:261]: I dont like this completely replacement the katana

File: code/game/objects/items/weapons/tools/simple_weapons.dm [hunk 2]
```diff
@@ -228,11 +228,11 @@
 //Swords
 
 /obj/item/tool/sword
-	name = "claymore"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/simple_weapons.dm:231]: I dont like this completely replacing the claymore

File: code/game/objects/items/weapons/tools/simple_weapons.dm (hunk 1 above)
Comment (thezblah) [reply code/game/objects/items/weapons/tools/simple_weapons.dm:261]: was given the go-ahead to resprite/name these as they are only obtainable by union crafting and are severely underused.

File: code/game/objects/items/weapons/tools/simple_weapons.dm (hunk 2 above)
Comment (thezblah) [reply code/game/objects/items/weapons/tools/simple_weapons.dm:231]: was given the go-ahead to resprite/name these as they are only obtainable by union crafting and are severely underused.

File: code/game/objects/items/weapons/tools/simple_weapons.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/weapons/tools/simple_weapons.dm:261]: This is a random spawn as well

File: code/game/objects/items/weapons/tools/simple_weapons.dm (hunk 2 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/weapons/tools/simple_weapons.dm:231]: This is a random spawn as well in loot pools

File: code/game/objects/items/weapons/tools/simple_weapons.dm (hunk 1 above)
Comment (thezblah) [reply code/game/objects/items/weapons/tools/simple_weapons.dm:261]: changing paths to the craftable one to sword/katana/union, adding a new entry for it, and restoring the original katana entry

File: code/game/objects/items/weapons/tools/simple_weapons.dm (hunk 2 above)
Comment (thezblah) [reply code/game/objects/items/weapons/tools/simple_weapons.dm:231]: changing paths to the craftable one to sword/union, adding a new entry for it, and restoring the original claymore entry

Comment (thezblah) [reply code/game/objects/items/weapons/tools/simple_weapons.dm:231]: changing paths to the craftable one to sword/union, adding a new entry for it, and restoring the original claymore entry

File: code/game/objects/items/weapons/tools/simple_weapons.dm [hunk 3]
```diff
@@ -273,6 +273,52 @@
...
+	item_state = "shamshir"
+	matter = list(MATERIAL_PLASTEEL = 15, MATERIAL_PLASTIC = 5)
+	sharp = TRUE
+	edge = TRUE
+	w_class = ITEM_SIZE_NORMAL
+	slot_flags = SLOT_BELT
+	worksound = WORKSOUND_HARD_SLASH
+	force = WEAPON_FORCE_ROBUST
+	armor_penetration = ARMOR_PEN_DEEP
+
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/simple_weapons.dm:290]:
```suggestion
	effective_faction = list("spider") // Spiders are weak to crushing.
	damage_mult = 1.5
```

File: code/game/objects/items/weapons/tools/simple_weapons.dm [hunk 4]
```diff
@@ -273,6 +273,52 @@
...
+	matter = list(MATERIAL_PLASTEEL = 10, MATERIAL_STEEL = 5, MATERIAL_DIAMOND = 1) //sharpened using diamond dust or whatever
+	slot_flags = SLOT_BELT | SLOT_BACK
+	force = WEAPON_FORCE_BRUTAL
+	armor_penetration = ARMOR_PEN_SHALLOW
+	item_icons = list(
+		slot_back_str = 'icons/inventory/back/mob.dmi')
+	item_state_slots = list(
+		slot_back_str = "katana_back"
+		)
+
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/simple_weapons.dm:321]:
```suggestion
	effective_faction = list("roach") // Spiders are weak to crushing.
	damage_mult = 1.5

```

**Comment by thezblah** [comment]
could the integration be failing because of rebel's union PR that got merged?

**Comment by thezblah** [comment]
ok i have no idea what happened but going back and committing trilby's 2nd suggestion (that I thought I committed before but didnt by accident) fixed the integration errors.  Databases are the devil and oh man am I not cut out to be an exorcist.

**Pull request merged**